### PR TITLE
fix: Add null value handler for time picker, fix latest time picker bugs

### PR DESCRIPTION
**Pull request by JKMarkowski**
#### Please provide a link to the associated issue.
fixes: https://github.com/SAP/fundamental-ngx/issues/2824
fixes: https://github.com/SAP/fundamental-ngx/issues/2900
#### Please provide a brief summary of this pull request.
There is 
- Removed console.log leftover,
- Fixed time elements height recognition inside popover, by adding `delay(0)` workaround
- Added null checks for examples. Time adaptation provided new way to handle invalid tie object. Now wrong object is sent outside. 

There is null value handler added for time picker 
timepickernew

#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/master/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist

Documentation checklist:
- [x] update `README.md`
- [x] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Comment by mikerodonnell89** [comment]
I think the "null validity" example should demonstrate this

**Comment by stefanoScalzo** [comment]
Not sure if invalid should keep the same time, also it should show something in the example that it is null

**Comment by InnaAtanasova** [comment]
Screen Shot 2020-07-23 at 4 07 57 PM

Is this ok?

**Comment by JKMarkowski** [comment]
@InnaAtanasova 
> Screen Shot 2020-07-23 at 4 07 57 PM
> 
> Is this ok?

DateTimePicker will be changed a little bit (validate on reactive forms). There are some differences in behaviour between Pickers, because of complexity.
This is also not scope of this PR.
datetimepicker

File: libs/core/src/lib/time-picker/time-picker.component.ts [hunk 1]
```diff
@@ -204,7 +206,12 @@ export class TimePickerComponent implements ControlValueAccessor, OnInit, AfterV
     handleIsOpenChange(isOpen: boolean): void {
         this.isOpen = isOpen;
         if (isOpen) {
-            this.child.changeActive('hour');
+            const subscription: Subscription = this.popover.directiveRef.loaded
+                .pipe(delay(0))
```
[the remark this answers is not in the document]
Comment (JKMarkowski) [reply libs/core/src/lib/time-picker/time-picker.component.ts]: Thanks for hint with `first` operator. I added takeUntil operator for destroy view.
Now hour is forced only for first open. You can see behaviour here:
timepicker1234

**Pull request merged**